frontend/pulse3d/src/components/interactiveAnalysis/InteractiveAnalysisModal.js

```diff
@@ -162,7 +162,18 @@ export default function InteractiveWaveformModal({ selectedJob, setOpenInteracti
   const [undoing, setUndoing] = useState(false);
   const [peakValleyWindows, setPeakValleyWindows] = useState({});
   const [duplicatesPresent, setDuplicatesPresent] = useState(false);
-
+  const [duplicateModalOpen, setDuplicateModalOpen] = useState(false);
+
+  const handleDuplicatesModalClose = (choice) => {
+    //0 = cancel
+    //1 = continue
+    if (choice === 0) {
+      setDuplicateModalOpen(false);
+    } else if (choice === 1) {
```

Comment:
The comment you left is a good indication that `choice` could just be renamed to one of the two possible options, `cancel` or `continue`. Also this Boolean logic should be refactored since `setDuplicateModalOpen(false);` is present in both (all) branches:
```
setDuplicateModalOpen(false);
if (continue) {
 postNewJob();
}
```